Resources/Audio/_Harmony/Weapons/Guns/Gunshots/attributions.yml

```
@@ -1,4 +1,4 @@
-- files: ["kammerer_shot.ogg"]
-  license: "CC-BY-4.0"
-  copyright: "Created by DrinkingWindGames, modified by halycon"
-  source: "https://freesound.org/people/DrinkingWindGames"
+- files: ["odin_fire.ogg"]
+  license: "CC-BY-NC-4.0"
+  copyright: "Individual sounds created by PNMCarrieRailfan and hyperix6, spliced together and modified by Halycon"
+  source: "https://freesound.org/people/PNMCarrieRailfan/"
```

That merge conflict was not resolved correctly, you completely replaced the previous version instead of adding the new sound to it.
```suggestion
- files: ["kammerer_shot.ogg"]
  license: "CC-BY-4.0"
  copyright: "Created by DrinkingWindGames, modified by halycon"
  source: "https://freesound.org/people/DrinkingWindGames"

- files: ["odin_fire.ogg"]
  license: "CC-BY-NC-4.0"
  copyright: "Individual sounds created by PNMCarrieRailfan and hyperix6, spliced together and modified by Halycon"
  source: "https://freesound.org/people/PNMCarrieRailfan/"

```